python/packages/aws-lambda-sdk/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=65.6.3",
+    "wheel>=0.38.4",
+]
+
+[project]
+name = "serverless-aws-lambda-sdk"
+version = "0.0.0"
+description = "Serverless AWS Lambda SDK for Python"
+authors = [{ name = "serverlessinc" }]
+requires-python = ">=3.7"
+dependencies = [
+    "serverless-sdk",
+    "serverless-sdk-schema",
```

Comment:
It'll be good to ensure latest of v0.1 to be installed (so if we publish breaking v0.2, we will need to upgrade it here as well)